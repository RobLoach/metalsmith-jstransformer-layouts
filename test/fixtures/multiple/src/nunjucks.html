---
layout: _nunjuckslayout.nunjucks
---
This is my site using Nunjucks!
