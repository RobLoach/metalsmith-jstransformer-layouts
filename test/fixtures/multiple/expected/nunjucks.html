<!DOCTYPE html>
<html>
  <body>This is my site using Nunjucks!
</body>
</html>
